[0m16:01:56.587112 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B085B5B650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B085B586E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B085B5A090>]}


============================== 16:01:56.591582 | c16553ef-68cf-48fe-8286-de98c29cf540 ==============================
[0m16:01:56.591582 [info ] [MainThread]: Running with dbt=1.7.9
[0m16:01:56.591582 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\benja\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'invocation_command': 'dbt init', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:02:46.579597 [debug] [MainThread]: Starter project path: C:\Users\benja\WCS\WCS_DBT_quests\my-dbt-env\Lib\site-packages\dbt\include\starter_project
[0m16:02:46.597686 [info ] [MainThread]: 
Your new dbt project "database" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:02:46.600770 [info ] [MainThread]: Setting up your profile.
[0m16:02:49.262362 [debug] [MainThread]: Command `dbt init` succeeded at 16:02:49.262362 after 52.77 seconds
[0m16:02:49.262362 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B0FF9166F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B0856D1370>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B085B1EC00>]}
[0m16:02:49.263389 [debug] [MainThread]: Flushing usage events
[0m16:58:45.182348 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8C4E0470>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8C32F320>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8C4E0140>]}


============================== 16:58:45.185914 | fd59d2dc-96ea-411c-a10a-38095dd8824a ==============================
[0m16:58:45.185914 [info ] [MainThread]: Running with dbt=1.7.9
[0m16:58:45.187451 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\benja\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:58:45.187451 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\benja\WCS\WCS_DBT_quests\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:58:45.189017 [debug] [MainThread]: Command `dbt run` failed at 16:58:45.189017 after 0.13 seconds
[0m16:58:45.189017 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8B9030B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8C4E0080>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000022A8C4E3CE0>]}
[0m16:58:45.190020 [debug] [MainThread]: Flushing usage events
[0m16:59:40.610018 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0F038F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0F03860>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0F037A0>]}


============================== 16:59:40.613666 | c5e14639-e840-4dfd-9f02-9fdda374a114 ==============================
[0m16:59:40.613666 [info ] [MainThread]: Running with dbt=1.7.9
[0m16:59:40.614640 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\benja\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m16:59:40.614640 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\benja\WCS\WCS_DBT_quests\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:59:40.616641 [debug] [MainThread]: Command `dbt run` failed at 16:59:40.616641 after 0.09 seconds
[0m16:59:40.616641 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0D53590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0F02930>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000195A0F02CF0>]}
[0m16:59:40.617641 [debug] [MainThread]: Flushing usage events
